client/src/components/SearchVideo.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+
+
+
+const SearchVideo = (prop) => {
+    let videos = prop.videos;
+  const [videoSearch, setvideo] = useState("");
```

Comment:
Remember to keep your state and state setter names the same for consistency

Reply:
done